crates/symbolicator/src/services/symbolication.rs

```diff
@@ -2036,10 +2033,10 @@ impl SymbolicationActor {
             };
 
             let missing_modules: Vec<(DebugId, &RawObjectInfo)> = result
-                .referenced_modules
+                .modules
                 .iter()
-                .filter(|(id, _)| !cfi_caches.has_module(id))
-                .map(|t| (t.0, &t.1))
+                .filter(|(id, _)| result.missing_ids.contains(id))
```

Comment:
instead of iterating over modules and filter by `missing_ids`, why not `into_iter` the `missing_ids`, and get the corresponding objectinfo from `modules`.

Reply:
actually. is it possible to get `missing_modules` (as in: id + metadata) directly, without the need to combine these two here?

Reply:
`missing_ids.into_iter()` would move out of `result`, which is still used later.

Reply:
> actually. is it possible to get missing_modules (as in: id + metadata) directly, without the need to combine these two here?

You mean put `missing_modules` as a field on `StackWalkMinidumpResult` and return it from `stackwalk_minidump_with_cfi`? Yeah, that should work.